kuma/settings/common.py

```diff
@@ -226,6 +226,7 @@ def __call__(self, value):
     'pt': 'pt-PT',
     'sr': 'sr-Cyrl',
     'zh': 'zh-CN',
+    'cn': 'zh-CN',
```

Comment:
nit: this should be at line 216, between ``bd`` and ``fy``, according to sorting rules

https://github.com/mozilla/kuma/blob/c6c6bb0ae8493d71fb77dd1d1948f2c051394720/kuma/settings/common.py#L214-L218